### DISCUSSION TITLE
Specified build config file does not exist: ./buildConfig/build.json

### DISCUSSION THREAD

**Discussion by donfire** [Sunbird-Ed/Community#21] [Installation] [answered]
Hi Team,
Getting the following errorr
```
[ERROR] An error occurred while running subprocess cordova.
        
        cordova build android --release --verbose --buildConfig ./buildConfig/build.json exited with exit code 1.
        
        Re-running this command with the --verbose flag may provide more information.
  ionic:utils-process onBeforeExit handler: 'process.exit' received +0ms
  ionic:utils-process onBeforeExit handler: running 2 functions +0ms
  ionic:utils-process processExit: exiting (exit code: 1) +145ms

```

**Comment by sharathkashyap** [comment, marked as answer]
@donfire You might have succesfully completed the build process. Better If there is complete Build Log to confirm the same. This Failure is mostly due to non availability of build.json which is not mandatory for local installation.
Kindly run this command to confirm the same. You should be able to have apk running.
`NODE_OPTIONS=--max-old-space-size=4096 ionic cordova run android`

This should complete apk creation.
